src/sentry/lang/javascript/errormapping.py

```diff
@@ -0,0 +1,95 @@
+import re
+import cgi
+import json
+import logging
+
+from django.conf import settings
+from django.core.cache import cache
+
+from sentry import http
+
+
+logger = logging.getLogger(__name__)
+
+
+error_processors = {}
+
+
+class Processor(object):
+
+    def __init__(self, vendor, mapping_url, regex, func):
+        self.vendor = vendor
+        self.mapping_url = mapping_url
+        self.regex = re.compile(regex)
+        self.func = func
+
+    def load_mapping(self):
+        key = 'javascript.errormapping:%s' % self.vendor
+        mapping = cache.get(key)
+        if mapping is not None:
+            return json.loads(mapping)
+
+        http_session = http.build_session()
+        response = http_session.get(self.mapping_url,
+            allow_redirects=True,
+            verify=False,
```

Comment:
Oh, but in this case, this is explicitly pulling from GitHub. They should have valid TLS certificate so I think it'd be more valuable to verify since we use this as a source for mappings.